app/controllers/chatrooms_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ChatroomsController < ApplicationController
+  
```

Comment:
This controller's action is available to all type of users, right? I meant about the authentication and the authorization. :D

Reply:
Yes, we develop authencation and the authorization late.
Right now, we have param "owner_id" in "User table". this param difine user, who owner chat room.